docs/sdk/examples/angular.html

```diff
@@ -51,7 +51,6 @@
 				outline: #2ba2f1 auto 5px;
 			}
 
-
 			.angular-nav__menu {
```

Comment:
Please add `padding: 0` here to remove small padding that is inherited from somewhere and isn't present in other frameworks samples.

Reply:
`padding: 0` is already here, the selector is not specific enough.